src/Resources/views/CRUD/Association/edit_many_to_many.html.twig

```diff
@@ -21,7 +21,11 @@ file that was distributed with this source code.
                                         {% if field_name == '_delete' %}
                                             <th>{{ 'action_delete'|trans({}, 'SonataAdminBundle') }}</th>
                                         {% else %}
-                                            <th {{ nested_field.vars['required']  ? 'class="required"' : '' }}>
+                                            <th
+                                                {% if (nested_field.vars['required'] is defined) and nested_field.vars['required'] %}
```

Comment:
How about using the `default` filter instead?

Reply:
Done